unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -163,6 +152,45 @@ export const EventDetails = ({
 
   const coverImage = event.ticket.event_cover_image
 
+  const renderEventLayout = () => {
+    switch (event.layout) {
+      case EventsLayoutEnum.Bannerless:
+        return (
+          <EventBannerlessLayout
+            event={event}
+            checkoutConfig={checkoutConfig}
+            hasLocation={hasLocation}
+            hasDate={hasDate}
+            startDate={startDate}
+            startTime={startTime}
+            endDate={endDate}
+            endTime={endTime}
+            eventUrl={eventUrl}
+            hasPassed={hasPassed}
+            organizers={organizers}
```

Comment:
Maybe put all these as a variable `layouProps = {...}` and then use `{...layoutProps}` so we can easily make sure we're passing them all and never forgetting any?